analyzers/src/SonarAnalyzer.CSharp/Common/Walkers/ParameterValidationInMethodWalker.cs

```diff
@@ -56,7 +56,7 @@ public override void VisitThrowStatement(ThrowStatementSyntax node)
                 && semanticModel.GetTypeInfo(node.Expression) is var typeInfo
                 && typeInfo.Type.DerivesFrom(KnownType.System_ArgumentException))
             {
-                argumentExceptionLocations.Add(node.Expression.GetLocation());
+                argumentExceptionLocations.Add(node.Expression.ToSecondaryLocation());
```

Comment:
The extension methods should all be named the same `ToSecondaryLocation` or `ToSecondary`

Reply:
This was discussed in the previous PR